src/main.cpp

```diff
@@ -68,20 +151,69 @@ int main() {
         }
         bot.getApi().sendMessage(message->chat->id, std::move(result));
     });
+    bot.getEvents().onCommand(
+        "get_friend_email", [&bot, &bot_commands, &users](TgBot::Message::Ptr message) {
```

Comment:
Name of a command is duplicated in a map and here.

Reply:
rewritten a bit